services/address_book_service.proto

```diff
@@ -29,37 +36,132 @@ import "transaction_response.proto";
 import "transaction.proto";
 
 /**
- * Transactions for the AddressBook Service, those HAPI APIs facilitate changes to the nodes used across the Hedera network.
- * All those transactions needs to be signed by the Hedera Council. Steps needed below:
- * 1. The node operator creates and signs the transaction with their key (the key on the node operator account)
- * 2. The node operator hands this transaction to Alex, who then gives it to the council to sign
- * 3. When signed and submitted, the server will verify that account 2 keys have signed, and the keys on the operator account have signed.
- * Hedera council should have ability to make all edits in addition to add/remove nodes
+ * The Address Book service provides the ability for Hedera network node
+ * administrators to add, update, and remove consensus nodes. This addition,
+ * update, or removal of a consensus node requires governing council approval,
+ * but each node operator may update their own operational attributes without
+ * additional approval, reducing overhead for routine operations.
```

Comment:
All updates need approval.

Reply:
Repeating my earlier response:
HIP text appears to claim a node operator can update their own node without council approval. Is council approval required for everything (in which case node operators are never actually required), or are node operators permitted to update their own nodes independently?

Reply:
From today's meeting, update don't need council approval.